pandas/core/groupby/groupby.py

```diff
@@ -489,7 +489,94 @@ class providing the base-class of operations.
 --------
 %(example)s"""
 
-_agg_template = """
+_agg_template_series = """
+Aggregate using one or more operations over the specified axis.
+
+Parameters
+----------
+func : function, str, list, dict or None
+    Function to use for aggregating the data. If a function, must either
+    work when passed a {klass} or when passed to {klass}.apply.
+
+    Accepted combinations are:
+
+    - function
+    - string function name
+    - list of functions and/or function names, e.g. ``[np.sum, 'mean']``
+    - dict of axis labels -> functions, function names or list of such.
```

Comment:
Should we remove noting that dicts can be passed since its deprecated?

Reply:
For most cases, my preference would be to have the User Guide reflect how we would like users to use pandas (so deprecations are to be removed) and the API docs to be an accurate reflection of the current arguments/behavior (so deprecations should not be removed).

However, in this case `dict` already doesn't work for SeriesGroupBy.agg when `as_index=True` (the default), so this one is a bit of a grey area. I'm good with removing here and will update.